src/context/WalletProvider/NativeWallet/components/EnterPassword.tsx

```diff
@@ -150,6 +150,7 @@ export const EnterPassword = () => {
             width='full'
             type='submit'
             isLoading={isSubmitting}
+            disabled={!isValid}
```

Comment:
Might be better UX? Now that we explicitly pass a `disabled` prop, we lose chakra's default disabled on loading
```suggestion
            disabled={!isValid || isSubmitting}
```

Reply:
Disregard me - this might work, but might as well use chakra's variant instead of this:

```suggestion
            isDisabled={!isValid}
```

Confirmed this achieves the same without the overhead